### PR TITLE
fix: pino prettyPrint

### PR DESCRIPTION
**Pull request by ndhuutai**
### Description of the Change

Modified loggers.ts to set prettyPrint to true for cli "service". Before the change, prettyPrint was not set to true (despite being called in init.ts in neo-one-cli) because init.ts was executed after loggers.ts (which creates different loggers for different environments.) 
Before:
Screen Shot 2021-03-04 at 12 26 25 PM
After:
Screen Shot 2021-03-04 at 2 42 10 PM

### Test Plan

1. Create symlinks for @neo-one/cli and @neo-one/smart-contract-typescript-plugin packages with 'yarn link'
2. Create and init a neo-one project. 
3. Apply the links to the above packages.
4. Intentionally add erroneous code.
5. Run yarn neo-one build.

For more details on symlinking check out [yarn docs on link](https://classic.yarnpkg.com/en/docs/cli/link/)

### Alternate Designs

N/A

### Benefits

Outputs better error message.

### Possible Drawbacks

N/A

### Applicable Issues

#2069

### CODE REVIEW

File: packages/neo-one-smart-contract-compiler/src/api.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ export const compileContract = async (
 ): Promise<CompileContractResult> => {
   const result = await compileContractBase({ filePath, host, linked });
 
-  throwOnDiagnosticErrorOrWarning(result.diagnostics, ignoreWarnings);
+  throwOnDiagnosticErrorOrWarning(result.diagnostics, ignoreWarnings, 'Error while compiling contract(s)');
```
Comment (spencercorwin) [inline packages/neo-one-smart-contract-compiler/src/api.ts:23]: Let's put the contract name here to make it a little cleaner. So it's `'Error while compiling contract ${contractName}'`

And change `_contractName` to `contractName`

File: packages/neo-one-smart-contract-compiler/src/utils/throwOnDiagnosticErrorOrWarning.ts [hunk 2]
```diff
@@ -2,7 +2,11 @@ import ts from 'typescript';
 
 import { getDiagnosticMessage } from './getDiagnosticMessage';
 
-export function throwOnDiagnosticErrorOrWarning(diagnostics: ReadonlyArray<ts.Diagnostic>, ignoreWarnings = false) {
+export function throwOnDiagnosticErrorOrWarning(
+  diagnostics: ReadonlyArray<ts.Diagnostic>,
+  ignoreWarnings = false,
+  message: string = 'There was an error',
```
Comment (spencercorwin) [inline packages/neo-one-smart-contract-compiler/src/utils/throwOnDiagnosticErrorOrWarning.ts:8]: Can we just make this an empty string? That seems cleaner

**Review by spencercorwin — CHANGES_REQUESTED** [review]
I'm not sure what happened with `neo-one-cli/bin/neo-one.js` but it shouldn't be empty

**Review by ndhuutai — COMMENTED** [review]
neo-one.js was untouched. The file still exists.

**Review by spencercorwin — APPROVED** [review]

**Pull request merged**